pkg/reconciler/workload/translate_test.go

```diff
@@ -241,7 +217,7 @@ func TestKubeAppWrapper(t *testing.T) {
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
 			r, err := KubeAppWrapper(context.Background(), tc.args.w, tc.args.o)
-
+			fmt.Println(name)
```

Comment:
Looks like this might have been from debugging? I suspect we should remove it or use `t.Log` instead.

Reply:
ah yes, thanks for catching!